nextflow/accession.nf

```diff
@@ -89,27 +91,28 @@ process create_properties {
  * Accession VCFs
  */
 process accession_vcf {
-    clusterOptions {
-        log_filename = accession_properties.getFileName().toString()
-        log_filename = log_filename.take(log_filename.indexOf('_accessioning.properties'))
-        return "-g /accession/instance-${params.instance_id} \
-                -o $params.logs_dir/accessioning.${log_filename}.log \
-                -e $params.logs_dir/accessioning.${log_filename}.err"
-    }
+    clusterOptions "-g /accession/instance-${params.instance_id} \
+                    -o $params.logs_dir/${log_filename}.log \
+                    -e $params.logs_dir/${log_filename}.err"
 
     memory '8 GB'
 
     input:
     path accession_properties from accession_props
     val accessioned_filename from accessioned_filenames
+    val log_filename from log_filenames
 
     output:
     path "${accessioned_filename}.tmp" into accession_done
 
     """
     filename=\$(basename $accession_properties)
     filename=\${filename%.*}
-    java -Xmx7g -jar $params.jar.accession_pipeline --spring.config.name=\$filename
+    java -Xmx7g -jar $params.jar.accession_pipeline --spring.config.name=\$filename || \
```

Comment:
Not necessarily related to this PR but something that we should consider in the future. Given the default amount of heap space that Java uses these days and the sizes of the files to be processed, 7GB might be a bit measly for large submissions. While this is fine as a default, I would suggest making this passable as a parameter.

Reply:
Good call, I've made EVA-2419 for this.